examples/simple-lambda.hs

```diff
@@ -40,3 +45,53 @@ code = "\
 \  callback(null, \"some success message\"); \
 \ } \
 \ "
+
+
+role :: Resource
+role = 
+  resource "IAMRole" $
+  IAMRoleProperties $
+  iamRole rolePolicyDocumentObject
+  & iamrPolicies ?~ [ executePolicy ]
+  & iamrRoleName ?~ "MyLambdaBasicExecutionRole"
+  & iamrPath ?~ "/"
+
+  where
+    executePolicy = 
+      iamPolicies
+      (HM.fromList
+      [ ("Version", "2012-10-17")
+      , ("Statement", statement) 
+      ]) $
+      "MyLambdaExecutionPolicy"
+      
+      where
+        statement = object
+          [ ("Effect", "Allow")
+          , ("Action", actions)
+          , ("Resource", "*")
+          ]
+
+        actions = Array $ V.fromList
```

Comment:
Again, you should be able to use `OverloadedLists` here. Though if you use the `aeson-qq` solution I shared then you don't need this vector :)

Also, if you absolutely need to add a dependency to an example, you can add it directly to the example's `dependencies` section in `package.yaml`. In this case, `vector` is totally innocuous (although not needed), but if you had to add a real dependency then it would be better to put it under the executable's section directly rather than globally.

Also note that the `IsList` class has a function [`fromList`](https://hackage.haskell.org/package/base-4.9.0.0/docs/GHC-Exts.html#v:fromList), so if something is an instance of `IsList` then you don't need to import that module just to get the `fromList` function. You can just say `fromList ...` and get a vector.